jitbuilder/build/files/common.mk

```diff
@@ -171,11 +171,6 @@ JIT_PRODUCT_BACKEND_SOURCES+=\
     $(JIT_OMR_DIRTY_DIR)/optimizer/ValueNumberInfo.cpp \
     $(JIT_OMR_DIRTY_DIR)/optimizer/VirtualGuardCoalescer.cpp \
     $(JIT_OMR_DIRTY_DIR)/optimizer/VirtualGuardHeadMerger.cpp \
-    $(JIT_OMR_DIRTY_DIR)/optimizer/VPConstraint.cpp \
```

Comment:
This is a duplicate, if you look for it in the entire file, you'll find two of them.